core/src/main/java/org/apache/spark/shuffle/sort/BypassMergeSortShuffleWriter.java

```diff
@@ -23,6 +23,7 @@
 import java.io.IOException;
 import javax.annotation.Nullable;
 
+import org.apache.spark.internal.config.package$;
```

Comment:
Import order should better be followed by https://github.com/databricks/scala-style-guide#imports

Reply:
ok, thanks